app/models/dictionary.rb

```diff
@@ -537,7 +537,13 @@ def expand_synonym
   end
 
   def synonym_expansion(synonyms)
-    # setting dummy method for next issue
+    expanded_synonyms = []
+    synonyms.each_with_index do |label, i|
+      expanded_label = "#{label}--dummy-synonym-#{i + 1}"
+      score = rand.round(4)
+      expanded_synonyms << { label: expanded_label, score: score }
+    end
+    expanded_synonyms
```

Comment:
```suggestion
    synonyms.map.with_index do |label, i|
      expanded_label = "#{label}--dummy-synonym-#{i + 1}"
      score = rand.round(4)
      { label: expanded_label, score:  }
    end
```

map.with_indexを使うと、変数expaded_synonymsが無くても良さそうです。

あとハッシュを作るときにバリューに設定する変数名がキーと同じ名前だと省略できます。
https://techracho.bpsinc.jp/hachi8833/2021_11_24/113383